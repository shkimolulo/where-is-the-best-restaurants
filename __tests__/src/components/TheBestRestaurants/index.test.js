import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';

describe('TheBestRestaurants', () => {
    let TheBestRestaurants;

    beforeEach(() => {
        TheBestRestaurants = require('../../../../src/components/TheBestRestaurants').default;  // mocking 하기 편하게 나온다
    });

    it('렌더링 시킬 수 있다.', () => {
        // Given

        // When
        const json = renderer.create(<TheBestRestaurants />).toJSON()
        // string 으로 만들어서

        // Then
        expect(json).toMatchSnapshot();
        // assert 와 같음

    }); // 하나의 테스트 단위
})
