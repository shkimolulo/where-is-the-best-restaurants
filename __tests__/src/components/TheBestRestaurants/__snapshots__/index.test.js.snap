// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TheBestRestaurants 렌더링 시킬 수 있다. 1`] = `
<View
  style={
    Object {
      "height": 1084,
      "marginTop": 50,
      "width": 750,
    }
  }
>
  <View />
</View>
`;
